entrypoint.sh

```diff
@@ -17,14 +30,19 @@ else
   VERSION='patch';
 fi
 
+# What is this? 41898282+github-actions[bot]@users.noreply.github.com
+# Configuring ... wat?
+# Bumping and pushing
+# Why pulling?
```

Comment:
pulling to update name of latest tag. git describe is not updated on push to remote, and gives you previous tag prior to pushing which is not what we want to make release from